docs/users/cli-usage.md

```diff
@@ -264,7 +249,29 @@ Your project can be reached at: http://drupal8.ddev.local
 
 And you can now visit your working project. Enjoy!
 
-_Please note that if you're providing the settings.php or wp-config.php and ddev is creating the settings.local.php (or wordpress wp-config-local.php), the main settings file must explicitly include the appropriate "settings.local.php" or equivalent._
+### Configuration files
+_**Note:** If you're providing the settings.php or wp-config.php and DDEV is creating the settings.ddev.php (or wp-config-local.php, AdditionalConfig.php, or similar), the main settings file must explicitly include the appropriate DDEV-generated settings file._
+
+The `ddev config` command attempts to create a CMS-specific settings file with DDEV credentials pre-populated.
+
+For **Drupal** and **Backdrop**, DDEV settings are written to a DDEV-managed file, settings.ddev.php. The `ddev config` command will ensure that these settings are included in your settings.php through the following steps:
+
+* Write DDEV settings to settings.ddev.php
```

Comment:
Small nit in that it doesn't just write, but it forces it to factory default each time.